target/linux/ramips/dts/mt7628an_360-p2.dts

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
```

Comment:
File name is wrong, hyphen -> "_". Has this not been tested?

Reply:
I got it, that's really a error here. I compiled OpenWrt v18.06 & v19.07 for this device some months ago and it can run well. I think it just a name error here. All gpio, port config and partition layout are totally right.